src/Filters/filt_order.jl

```diff
@@ -0,0 +1,458 @@
+                (Wa, Ωpadj) = $(Symbol(string(est, "_bsfmin")))(Ωp, Ωs, Rp, Rs) # check scipy.
+            end
+            N = ceil(Int, $(Symbol(string(est, "_order_estimate")))(Rp, Rs, minimum(abs.(Wa))))
+            ωn = (domain == :z) ? Wps : Ωpadj
+            N, ωn
+        end
+    end
+end
+
+
+"""
+    (N, ωn) = cheb2ord(Wp::Real, Ws::Real, Rp::Real, Rs::Real; domain::Symbol=:z)
+
+Integer and natural frequency order estimation for Chebyshev Type II (inverse) Filters. `Wp` and `Ws`
+are the frequency edges for Bandpass/Bandstop cases, with `Rp` and `Rs` representing the ripple maximum
+loss in the passband and minimum ripple attenuation in the stopband in dB. Based on the ordering of
+the passband and stopband edges, the Lowpass or Highpass filter type is inferred. `N` is the integer
+indicating the lowest filter order, with `ωn` specifying the "-3 dB" cutoff frequency. If the domain
+is specified as `:s`, the passband and stopband frequencies are interpretted as radians/second, giving
+the order and natural frequencies for an analog filter. The default domain is `:z`, interpretting the
+input frequencies as normalized from 0 to 1, where 1 corresponds to π radians/sample.
+"""
+function cheb2ord(Wp::Real, Ws::Real, Rp::Real, Rs::Real; domain::Symbol=:z)
+    ftype = (Wp < Ws) ? Lowpass : Highpass
+    (Ωp, Ωs) = (domain == :z) ? (tan(π/2 * Wp), tan(π/2 * Ws)) : (Wp, Ws)
+    wa = toprototype(Ωp, Ωs, ftype)
+    N = ceil(Int, chebyshev_order_estimate(Rp, Rs, wa))
+
+    # new frequency for stopband spec.
+    wnew = 1 / cosh(1 / N * acosh(√(db2pow(Rs) - 1) / √(db2pow(Rp) - 1)))
+    wa = (ftype == Lowpass) ? Ωp / wnew : Ωp * wnew
+    ωn = (domain == :z) ? (2/π)*atan(wa) : wa
+    N, ωn
+end
+
+
+"""
+    (N, ωn) = cheb2ord(Wp::Tuple{Real, Real}, Ws::Tuple{Real, Real}, Rp::Real, Rs::Real; domain::Symbol=:z)
+
+Integer and natural frequency order estimation for Chebyshev Type II (inverse) Filters. `Wp` and `Ws` are 
+2-element frequency edges for Bandpass/Bandstop cases, with `Rp` and `Rs` representing  the ripple maximum 
+loss in the passband and minimum ripple attenuation in the stopband in dB. Based on the ordering of passband 
+and bandstop edges, the Bandstop or Bandpass filter type is inferred. `N` is an integer indicating the 
+lowest estimated filter order, with `ωn` specifying the cutoff or "-3 dB" frequencies. If a domain of 
+`:s` is specified, the passband and stopband frequencies are interpretted as radians/second, giving an 
+order and natural frequencies for an analog filter. The default domain is `:z`, interpretting the input 
+frequencies as normalized from 0 to 1, where 1 corresponds to π radians/sample.
+"""
+function cheb2ord(Wp::Tuple{Real, Real}, Ws::Tuple{Real, Real}, Rp::Real, Rs::Real; domain::Symbol=:z)
+    Wps = (Wp[1] > Wp[2]) ? tuple(Wp[2], Wp[1]) : tuple(Wp[1], Wp[2])
+    Wss = (Ws[1] > Ws[2]) ? tuple(Ws[2], Ws[1]) : tuple(Ws[1], Ws[2])
+    ftype = (Wps[1] < Wss[1]) ? Bandstop : Bandpass
```

Comment:
And here, too: Should we verify that `(Wps[1] < Wss[1]) == (Wps[2] > Wss[2])` and throw an informative error otherwise?